contracts/sei-tester/src/contract.rs

```diff
@@ -10,8 +10,9 @@ use crate::msg::{
 use sei_cosmwasm::{
     DexTwapsResponse, EpochResponse, ExchangeRatesResponse, GetOrderByIdResponse,
     GetOrdersResponse, OracleTwapsResponse, Order, OrderSimulationResponse, OrderType,
-    PositionDirection, SeiMsg, SeiQuerier, SeiQueryWrapper,
+    PositionDirection, SeiMsg, SeiQuerier, SeiQueryWrapper, 
 };
+use vortex_common::types::PositionEffect;
```

Comment:
Why are we importing vortex stuff here? Ideally the tester contract is agnostic to individual product contracts. If this struct would be needed by multiple contracts then we should instead define it in sei cosmwasm and import that in vortex imo.

Reply:
good call, will separate vortex stuff with sei-cosmwasm